core/unit_test/TestRange.hpp

```diff
@@ -327,6 +327,61 @@ TEST(TEST_CATEGORY, range_reduce) {
   }
 }
 
+template <typename ExecSpace, typename StaticBatchSize>
+struct TestStaticBatchSize {
+  using view_type = Kokkos::View<int *, ExecSpace>;
+
+  view_type m_flags;
+  view_type result_view;
+
+  struct AtomicAddTag {};
+  struct VerifyAtomicAddTag {};
+
+  int N;
+
+  TestStaticBatchSize(const size_t N_)
```

Comment:
Make sure your types agree.
```suggestion
  size_t N;

  TestStaticBatchSize(const size_t N_)
```